doc/users/whats_new/reproducible_ps_pdf.rst

```diff
@@ -0,0 +1,6 @@
+Reproducible PS and PDF output
+------------------------------
+
+The ``SOURCE_DATE_EPOCH`` environment variable can now be used to set
+the timestamps value in the PS and PDF outputs, which are then
+reproducible.
```

Comment:
We should list the known limitations, because when downstream users see this promise, they may start depending on it and run into the limitations later. At least usetex with the ps backend should be ruled out, based on comments in the tests.